src/web/app/app.tsx

```diff
@@ -252,6 +284,58 @@ export function FilterCheckbox(props: {
   );
 }
 
+export function FilterPriority(props: {
```

Comment:
nit: it is a filter ui control, so name it `PriorityFilter` (same with the search filter below)

Reply:
Done.